<a href="https://colab.research.google.com/github/Jos-lab02/Solucion-introduccion-python/blob/main/01_IntroPython__Gua%C3%B1unaJose_ipynb_.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [ ]:
import pandas as pd
df = pd.read_csv("titanic.csv")
df.describe()
df.head()
print("Jose Guañuna")
Porcentaje_Supervivencia= df["Survived"].mean()*100
print(f"\nPorcentaje de Supervivencia: {Porcentaje_Supervivencia:.2f}%")
Porcentaje_Clase= df["Pclass"].value_counts(normalize=True)*100
print("Porcentaje de Clase")
for clase, valor in Porcentaje_Clase.items():
    print(f"Clase {clase}: {valor:.2f}%")
print("Cantidad por genero")
Porcentaje_Sexo= df["Sex"].value_counts()
for genero, cantidad in Porcentaje_Sexo.items():
    print(f"{genero}: {cantidad}")

Jose Guañuna

Porcentaje de Supervivencia: 38.38%
Porcentaje de Clase
Clase 3: 55.11%
Clase 1: 24.24%
Clase 2: 20.65%
Cantidad por genero
male: 577
female: 314


En conclusion se puede evidenciar que el porcentaje de sobrevivientes fue tan solo del 38.38% , lo cual demuestra que el titanic fue una tragedia muy grande , ademas se evidencia que la mayor parte de gente que iba en este barco era de tercera clase , seguido de personas de primera clase y por ultimo personas de segunda clase dando a entender que la mayoria de personas talvez trabajaban ahi y eran de clase 3 mientras que los pasajeros en su mayoria serian de clase 1 y 2
Finalmente la cantidad de hombres y mujeres no nos dice mas que la cantidad de cada genero dentro del bote ,al evidenciar que la mayor cantidad eran hombres se podria intuir que la mayor concentracion de trabajadores y gente de clase alta eran hombre , sin embargo sin mas informacion solo son especulaciones